{"uid":"5843ee57967d667c","name":"test_delete_message","fullName":"test_Front.TestMessage#test_delete_message","historyId":"2497bcad1cba37512341cf8dbcca0d4d","time":{"start":1729945446419,"stop":1729945446419,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nStacktrace:\n#0 0x5634d49480aa <unknown>\n#1 0x5634d445f1a0 <unknown>\n#2 0x5634d44abd06 <unknown>\n#3 0x5634d44abfa1 <unknown>\n#4 0x5634d44f1344 <unknown>\n#5 0x5634d44cfdad <unknown>\n#6 0x5634d44ee866 <unknown>\n#7 0x5634d44cfb23 <unknown>\n#8 0x5634d449e990 <unknown>\n#9 0x5634d449f96e <unknown>\n#10 0x5634d491416b <unknown>\n#11 0x5634d4917f68 <unknown>\n#12 0x5634d490164c <unknown>\n#13 0x5634d4918ae7 <unknown>\n#14 0x5634d48e64af <unknown>\n#15 0x5634d49364f8 <unknown>\n#16 0x5634d49366c0 <unknown>\n#17 0x5634d4946f26 <unknown>\n#18 0x7f5af4694ac3 <unknown>","statusTrace":"request = <SubRequest 'init_driver' for <Function test_send_message>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_driver(request):\n        # Инициализация драйвера\n        options = webdriver.ChromeOptions()\n        options.add_argument('--headless')\n        driver = webdriver.Chrome(options=options)\n    \n        driver.implicitly_wait(10)\n        driver.maximize_window()\n        request.cls.driver = driver\n    \n        # Инициализация страницы\n        login_page = LoginPage(driver)\n        home_page = HomePage(driver)\n    \n        # Открытие страницы\n        login_page.open_login_page()\n    \n        # Логинимся\n>       login_page.login_user()\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nlogin_page.py:27: in login_user\n    self.find_element(self.login_user_button_locator())\nbase_page.py:13: in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cfd06bb5501f93f6e8d5a53f0dbe2974\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f84a0295870>\nmessage = 'Can\\'t find element by locator (\\'xpath\\', \\'//button//div[text()=\"Вход\"]\\')'\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nE       Stacktrace:\nE       #0 0x5634d49480aa <unknown>\nE       #1 0x5634d445f1a0 <unknown>\nE       #2 0x5634d44abd06 <unknown>\nE       #3 0x5634d44abfa1 <unknown>\nE       #4 0x5634d44f1344 <unknown>\nE       #5 0x5634d44cfdad <unknown>\nE       #6 0x5634d44ee866 <unknown>\nE       #7 0x5634d44cfb23 <unknown>\nE       #8 0x5634d449e990 <unknown>\nE       #9 0x5634d449f96e <unknown>\nE       #10 0x5634d491416b <unknown>\nE       #11 0x5634d4917f68 <unknown>\nE       #12 0x5634d490164c <unknown>\nE       #13 0x5634d4918ae7 <unknown>\nE       #14 0x5634d48e64af <unknown>\nE       #15 0x5634d49364f8 <unknown>\nE       #16 0x5634d49366c0 <unknown>\nE       #17 0x5634d4946f26 <unknown>\nE       #18 0x7f5af4694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"init_driver","time":{"start":1729945432444,"stop":1729945446380,"duration":13936},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nStacktrace:\n#0 0x5634d49480aa <unknown>\n#1 0x5634d445f1a0 <unknown>\n#2 0x5634d44abd06 <unknown>\n#3 0x5634d44abfa1 <unknown>\n#4 0x5634d44f1344 <unknown>\n#5 0x5634d44cfdad <unknown>\n#6 0x5634d44ee866 <unknown>\n#7 0x5634d44cfb23 <unknown>\n#8 0x5634d449e990 <unknown>\n#9 0x5634d449f96e <unknown>\n#10 0x5634d491416b <unknown>\n#11 0x5634d4917f68 <unknown>\n#12 0x5634d490164c <unknown>\n#13 0x5634d4918ae7 <unknown>\n#14 0x5634d48e64af <unknown>\n#15 0x5634d49364f8 <unknown>\n#16 0x5634d49366c0 <unknown>\n#17 0x5634d4946f26 <unknown>\n#18 0x7f5af4694ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/Dc/Dc/conftest.py\", line 29, in init_driver\n    login_page.login_user()\n  File \"/home/runner/work/Dc/Dc/login_page.py\", line 27, in login_user\n    self.find_element(self.login_user_button_locator())\n  File \"/home/runner/work/Dc/Dc/base_page.py\", line 13, in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true},{"name":"base_url","time":{"start":1729945432444,"stop":1729945432444,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('init_driver', 'base_url')"},{"name":"suite","value":"test_Front"},{"name":"subSuite","value":"TestMessage"},{"name":"host","value":"fv-az775-215"},{"name":"thread","value":"2214-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_Front"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.usefixtures('init_driver', 'base_url')"]},"source":"5843ee57967d667c.json","parameterValues":[]}